selenium/che-selenium-test/src/test/resources/templates/workspace/docker/default_ws_with_predefined_projects.json

```diff
@@ -0,0 +1,73 @@
+{
+  "environments": {
+    "replaced_name": {
+      "machines": {
+        "dev-machine": {
+          "installers": [
+            "org.eclipse.che.ws-agent"
+          ],
+          "attributes": {
+            "memoryLimitBytes": "2147483648"
```

Comment:
It's better to have an ability to set "desired_memory_value" automatically through injection.